features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,635 @@
+      | geslachtsaanduiding (04.10)          | M           |
+      | voornamen (02.10)                    | Hendrik Jan |
+      | adellijke titel of predicaat (02.20) | R           |
+      | voorvoegsel (02.30)                  | de          |
+      | geslachtsnaam (02.40)                | Brink       |
+      | aanduiding naamgebruik (61.10)       | E           |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                 | waarde    |
+      | burgerservicenummer (01.20)          | 000000188 |
+      | geslachtsaanduiding (04.10)          | V         |
+      | voornamen (02.10)                    | Carolina  |
+      | adellijke titel of predicaat (02.20) | BS        |
+      | voorvoegsel (02.30)                  | van der   |
+      | geslachtsnaam (02.40)                | Horn      |
+
+  Rule: wanneer gevraagd wordt om voorletters, wordt voornamen geleverd
+
+    Scenario: met fields is gevraagd om voorletters
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000176                       |
+      | fields              | naam.voorletters                |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam           | waarde      |
+      | naam.voornamen | Hendrik Jan |
+
+    Scenario: met fields is gevraagd om de voorletters van de partner
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000176                       |
+      | fields              | partners.naam.voorletters       |
+      Dan heeft de response een persoon met een 'partner' met de volgende gegevens
+      | naam           | waarde   |
+      | naam.voornamen | Carolina |
+
+  Rule: wanneer gevraagd wordt om volledigeNaam, wordt voornamen, adellijkeTitelPredicaat, voorvoegsel, geslachtsnaam en aanduidingNaamgebruik geleverd
+    
+    Scenario: met fields is gevraagd om volledigeNaam
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000176                       |
+      | fields              | naam.volledigeNaam              |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                                 | waarde                        |
+      | voornamen                            | Hendrik Jan                   |
+      | adellijkeTitelPredicaat.code         | R                             |
+      | adellijkeTitelPredicaat.omschrijving | Ridder                        |
```

Comment:
dit wordt lowercase als de landelijke tabellen worden geïmporteerd. Wil je dat in de p.r. al meenemen?